helm_drydock/config.py

```diff
@@ -21,13 +21,24 @@
 
 class DrydockConfig(object):
 
+    global_config = {
+        'log_level':    'DEBUG',
+    }
+
     node_driver = {
         'maasdriver':   {
             'api_key':  'KTMHgA42cNSMnfmJ82:cdg4yQUhp542aHsCTV:7Dc2KB9hQpWq3LfQAAAKAj6wdg22yWxZ',
```

Comment:
It isn't and I've already deleted this key and created a new one. A near future PR will support config files and there will be no hard coded config values in the code base.